use adafruit_anchored_group instead of sized_group

FoamyGuy opened the pull request:
and add bitmap files to repo

adafruit_anchored_group has a release and open PR adding to the bundle: https://github.com/adafruit/Adafruit_CircuitPython_Bundle/pull/501

tannewt reviewed (changes requested):
Please put the bmp files in a folder. That'll keep the root less program specific.

FoamyGuy commented:
I Moved them into `launcher_assets` and added a default_icon which was also missing.  I made a new one that uses fewer colors.

Also removed sized_group.py which isn't used any longer.
